packages/ember-htmlbars/lib/components/link-to.js

```diff
@@ -758,10 +758,20 @@ let LinkComponent = EmberComponent.extend({
   willRender() {
     let queryParams;
 
-    // Do not mutate params in place
-    let params = get(this, 'params').slice();
+    let params = get(this, 'params');
```

Comment:
Based on how `processedArgs` for `positionalParams` in glimmer we must guard here.